test/e2e/app-dir/head/head.test.ts

```diff
@@ -41,9 +41,24 @@ createNextDescribe(
     })
 
     it('should use head from layout when not on page', async () => {
+      const errors = []
+      next.on('stderr', (args) => {
+        errors.push(args)
+      })
+
       const $ = await next.render$('/blog/about')
       const headTags = $('head').children().toArray()
 
+      if (globalThis.isNextDev) {
+        expect(
```

Comment:
This should probably use `check()` as the logs may not be instantly available to check.